public/.htaccess.dist

```diff
@@ -83,4 +83,10 @@ DirectoryIndex index.php
   </IfModule>
 </IfModule>
 
+# Uncomment if you use folder protection via .htaccess/.htpasswd in accordance to documentation
```

Comment:
I would remove `in accordance to documentation` or link to a documentation page 😅